README.md

```diff
@@ -2,18 +2,20 @@
 
 # Project Name
 
-> in this project we implemented modular workflow optimization tools in our projects such as webpack, created a page dynamically with JS and styled it with CSS.
-
+> In this project, I implemented modular workflow optimization tools such as webpack, created a page dynamically with JS, and styled it with CSS.
 
 ## Built With
 
-- Webpack
-- JS HTML CSS
+- Webpack.
+- JS HTML CSS.
 
 ## Live Demo (if available)
 [Live Demo Link](https://livedemo.com)
+
+
 ## Getting Started
 
+
```

Comment:
- [ ] @diegoh40  as indicated earlier kindly remove the unnecessary headings in your getting section eg prerequisites, setup install. You could also choose to finish the section by adding content to your getting started section.